docs/en/05_Contributing/04_Release_Process.md

```diff
@@ -33,34 +36,98 @@ announced on the ["releases" forum category](https://forum.silverstripe.org/c/re
 
 ## Release numbering {#release-numbering}
 
-SilverStripe follows [Semantic Versioning](http://semver.org).
+Silverstripe CMS follows [Semantic Versioning](http://semver.org).
 
-All SilverStripe modules (including silverstripe/framework) may have patch releases created at any time, and will
+All Silverstripe CMS modules (including `silverstripe/framework`) may have patch releases created at any time, and will
 not necessarily match other core module patch release numbers. For example, your project may be using
-silverstripe/cms 4.3.1 with silverstripe/versioned 4.3.9.
+`silverstripe/cms` 4.3.1 with `silverstripe/versioned` 4.3.9.
 
-All SilverStripe recipes are released in lock step with each other. For example, silverstripe/installer 4.3.1 will
-contain silverstripe/recipe-cms 4.3.1 and silverstripe/recipe-core 4.3.1. These recipes may contain various patch
+All Silverstripe CMS recipes are released in lock step with each other. For example, `silverstripe/installer` 4.3.1 will
+contain `silverstripe/recipe-cms` 4.3.1 and `silverstripe/recipe-core` 4.3.1. These recipes may contain various patch
 versions of its modules within the same minor release line (4.3 in this example).
 
+## Regular quarterly releases
+
+A regular Silverstripe CMS minor release is tagged every quarter. This involves creating a new minor increment of 
```

Comment:
It doesn't have to be a `minor`. E.g. we have skept releasing a minor `4.5` in June and did a bunch of security patch releases instead (which was 4.4.4, 4.3.5, 4.2.5, 3.7.4 and 3.6.8).

Should we rephrase to mention that it's `usually a minor, but doesn't have to be`?

Reply:
```suggestion
A regular Silverstripe CMS release is tagged every quarter. This usually involves creating a new minor increment of 
```